jsapp/js/account/organization/MemberRemoveModal.tsx

```diff
@@ -0,0 +1,118 @@
+// Partial components
+import Button from 'jsapp/js/components/common/button';
+import InlineMessage from 'jsapp/js/components/common/inlineMessage';
+import KoboModal from 'jsapp/js/components/modals/koboModal';
+import KoboModalHeader from 'jsapp/js/components/modals/koboModalHeader';
+import KoboModalContent from 'jsapp/js/components/modals/koboModalContent';
+import KoboModalFooter from 'jsapp/js/components/modals/koboModalFooter';
+
+// Stores, hooks and utilities
+import {getSimpleMMOLabel} from './organization.utils';
+import envStore from 'jsapp/js/envStore';
+import subscriptionStore from 'jsapp/js/account/subscriptionStore';
+import {useRemoveOrganizationMember} from './membersQuery';
+import {notify} from 'alertifyjs';
+
+export const REMOVE_SELF_TEXT = {
+  title: t('Leave this ##team/org##'),
```

Comment:
I think ##TEAM_OR_ORGANIZATION## won your recent poll, so should we change these?